src/common/functions/replaceSpace.ts

```diff
@@ -0,0 +1,2 @@
+export const replaceSpace = (text: string, replaceStr = '_'): string =>
```

Comment:
can we rename `functions` to `utils`?

Reply:
changed here https://github.com/keypom/keypom-airfoil/pull/52/commits/3d68f96d608c47c7c35ee2bea3d53bcf19efea2f